community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/csv/InputEntityDeserializer.java

```diff
@@ -176,6 +186,11 @@ private boolean deserializeNextFromSource() throws IOException
     @Override
     public void close()
     {
+        if ( ignoredColumns > 0 )
+        {
+            System.out.println( String.format( "Warning: Ignored %d extra columns in %s", ignoredColumns,
```

Comment:
I'd prefer to get a histogram output of the ignored columns.

Something like:
```
Warning: Ignored 2 extra columns in 4 lines of /your/file.csv
Warning: Ignored 5 extra columns in 3 lines of /your/file.csv
```

Maintaining that state is easy enough, and it helps immensely in figuring out what the reason for the extra columns might be. Especially if we also print the total number of lines of the file.